### ISSUE TITLE
Bootstrap Test, (from PC)

### ISSUE THREAD

**Issue by maccer78** [maccer78/Bootstrap-Test#1]
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Bootstrap Website Tutorial</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  <style>
  .navbar {
    margin-bottom: 0;
    border-radius: 0;
    background-color: #5E4485; /* adjusting the color scheme of navbar */
    color: #FFF;
    padding: 1% 0;
    font-size: 1.2em;
    border:0;
  }
  .navbar-brand {
    float: left;
    min-height: 55px;
    padding: 0 15px 5px; /* FLoats the w3 logo up */
  }
  .navbar-inverse .navbar-nav .active a, .navbar-inverse .navbar-nav .active a:focus, .navbar-inverse .navbar-nav .active a:hover, {
    color: #FFF;
    background-color: #5E4485;
  }
  .navbar-inverse .navbar-nav li a {
    color: #D5D5D5;
  }
  .carousel-caption {
    top:50%;
    transform: translateY(-50%);
    text-transform: uppercase;
  }

    .btn {
      font-size: 18px;
      color: #FFF;
      padding: 12px 22px;
      background: #5E4485;
      border: 2px solid #FFF; /* Adding Color and border */
    }
  @media (max-width:600px) {
    .carousel caption {
      display: none;
    }
  }
  .container {
  margin: 4% auto;
  }
  #icon {
  max-width: 200px; /* Adjusting the icon size */
  margin: 1% auto;
  }
  .fill {
  width: 100%;
  height: 100%;
  background-position: center;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  background-size: cover;
  -o-background-size: cover;
  }
  h1 {
  text-shadow:1px 1px 1px rgba(18,1,1,1);
  font-style:oblique;
  color:#FFF;
  letter-spacing:1pt;
  word-spacing:9pt;
  font-size: 60px;
  text-align:left;
  text-align: center;
  }

  </style>
</head>
<body>
  <nav class="navbar navbar-inverse">
    <div class="container-fluid">
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" date-toggle="collapse" date-target="#myNavbar">
          <span class="icon bar"> </span>
          <span class="icon bar"> </span>
          <span class="icon bar"> </span>
        </button>
        <a class="navbar-brand" href="#"><img src="img/w3newbie.png"></a>
      </div>
      <div class="collapse navbar-collapse" id="myNavbar">
        <ul class="nav navbar-nav navbar-right">
          <li class="active"><a href="#">Home</a><li>
          <li><a href="#">About</a></li>
          <li><a href="#">Services</a></li>
          <li><a href="#">Testinomials</a></li>
          <li><a href="#">Contact</a></li>

        </ul>
      </div>
    </div>
  </nav>
   <div id="myCarousel" class="carousel slide" data-ride="carousel">         
    <ol class="carousel-indicators">
      <li data-target="#myCarousel" data-slider-to="0" class="active"></li>
      <li data-target="#myCarousel" data-slider-to="1"></li>
      <li data-target="#myCarousel" data-slider-to="2"></li>
    </ol>
    <div class="carousel-inner" role="listbox">
      <div class="item active">
        <img class="fill" src="img/mountains.png">
        <div class="carousel-caption">
          <h1>get to know bootstrap</h1>
          <br>
          <button type="button" class="btn btn-defualt">Get Started</button>
      </div>
    </div> 
    <div class="item"> 
      <img class="fill" src="img/snow.png">
    </div>
    <div class="item">
      <img class="fill" src="img/red.png">
    </div>
  </div>
  
    <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
      <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
      <span class="sr-only">Previous</span>
    </a>
    <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
      <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
      <span class="sr-only">Next</span>
    </a>
</div>
 

  <div class="container text-center">
    <h2>What We're Using</h2>
    <div class="row">
      <div class="col-sm-4">
        <img src="img/html5.png" id="icon">
        <h4> HTML5 </h4>
      </div>
      <div class="col-sm-4">
        <img src="img/bootstrap.png" id="icon">
          <h4> Bootstrap </h4>
      </div>
      <div class="col-sm-4">
        <img src="img/Css3.png" id="icon">
          <h4> CSS3 </h4>
      </div>
    </div>
  </div>
    <div class="container">
      <div class="row">
        <div class="col-md-6">
          <h4> Here's the cool thing about bootstrap...</h4>
          <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit,
            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
             ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
             voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
             sunt in culpa qui officia deserunt mollit anim id est laborum."
          </p>
          <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit,
            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
             ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
             voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
             sunt in culpa qui officia deserunt mollit anim id est laborum."
        </div>
        <div class="col-md-6">
          <img src="img/bootstrap2.jpg" class="img-responsive">
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-lg-3 col-sm-6 col-xs-12">
          <h4> Built With Sass</h4>
          <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit,
            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
             ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
             voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
             sunt in culpa qui officia deserunt mollit anim id est laborum."
          </p>
        </div>
        <div class="col-lg-3 col-sm-6 col-xs-12">
          <img src="img/sass.png" class="img-responsive">
        </div>
        <div class="col-lg-3 col-sm-6 col-xs-12">
          <h4> And Less.</h4>
          <p>"Lorem ipsum dolor sit amet, consecte tur adipiscing elit,
            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
             ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
             voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
             sunt in culpa qui officia deserunt mollit anim id est laborum."
          </p>
        </div>
        <div class="col-lg-3 col-sm-6 col-xs-12">
          <img src="img/less.png" class="img-responsive">
        </div>
      </div>
    </div>

</body>
</html>
